vuepress/docs/next/tutorials/devops/change-default-datasource.md

```diff
@@ -1,74 +1,53 @@
-#### 1.1. The standalone.xml/standalone-openshift.xml file of choice 
-It is essential for your custom Docker image to use a file from one of the official Entando Docker base images.
+- A running instance of Entando with the required external databases. [Install Entando on any Kubernetes provider](../#operations) or see [Getting Started](../../docs/getting-started/) for more information. 
 
-If you intend to provide your own EAP-based image, please use this 
-[standalone-openshift.xml](https://github.com/entando/entando-app-engine-base/blob/develop/eap/standalone-openshift.xml)
-file. 
+- Your own custom project as described in [this tutorial](./build-core-image.md) 
 
-If you intend to provide your own Wildfly-based image, please use this 
-[standalone.xml](https://github.com/entando/entando-app-engine-base/blob/develop/wildfly/contrib/wildfly-configuration/standalone.xml)
-file.
+## Customize the Docker Image
+Create a Docker project for your base image using Entando's standard base image. Add the following files to the project folder, typically to a repository like GitHub.  
 
-Please note that if you create your own base image with a different version of EAP or Wildfly, using these files 
-may have unintended consequences.
+**Is server.xml the correct one?**
+1. Using the following Entando template for the Docker image, create the `server.xml`. 
+ 
+If you intend to provide your own Tomcat-based image, please use this 
+[server.xml](https://github.com/entando/entando-app-engine-base/blob/develop/tomcat/conf/server.xml)  
+file.  
 
-#### 1.2. Any modules that may be required for your datasource or other connection resource
-If the resource you need to connect to requires some custom classes, such as JDBC drivers, please add these as  
-a module to Wildfly/EAP. 
+Please note that creating your own base image with a different version of Tomcat may have unintended consequences.
 
-#### 1.3 Dockerfile
-In your Dockerfile, please be sure to extend the correct base image, and add the correct configuration file to
-the correct location in the target image. 
+2. Add any modules that may be required for your datasource or other connection resource. If the resource you need to connect to requires custom classes such as JDBC drivers, please add these as  
+a module to the Tomcat image. 
 
-For EAP, a most basic Dockerfile would look like this:
+3. In your **Dockerfile** for the new project, be sure to extend the correct base image and add the correct configuration file to the correct location. 
 
+For Tomcat, a basic Dockerfile would look like this:
+**Need new address & names**  
 ```
-FROM entando/entando-eap71-clustered-base:6.1.2
-COPY --chown=185:0 ./standalone-openshift.xml /opt/eap/standalone/configuration
-```  
- 
-For Wildfly, a most basic Dockerfile would look like this:
+FROM entando/entando-XXXX-clustered-base:6.1.2
+COPY --chown=185:0 ./XXX.xml /opt/XXX/standalone/configuration
```

Comment:
@pietrangelo @nshaw seems like this is a sample we provide for them to look at, do we have a replacement for the tomcat version?

Reply:
The reference version is here - https://github.com/entando/app-engine/blob/develop/Dockerfile.tomcat - and has this line  FROM registry.hub.docker.com/entando/entando-tomcat-base:7.2.0